packages/ui/src/components/Checkbox.tsx

```diff
@@ -1 +1,23 @@
-export { Checkbox, type CheckboxProps } from 'theme-ui';
+import {
+  type CheckboxProps as CheckboxPropsUi,
+  Checkbox as CheckboxUi,
+} from 'theme-ui';
+import React from 'react';
+
+export type CheckboxProps = CheckboxPropsUi & {
```

Comment:
I prefer not to add these props to the UI element. I prefer to keep their API as simple as possible.

Instead of passing `error,` just pass `aria-invalid`

Reply:
Got it, I'm gonna update it